app/styles/variables.css

```diff
@@ -1,7 +1,7 @@
 :root {
   --color-white: #ffffff;
   --color-offwhite: #fffbfb;
-  --color-greywhite: #d0e7ff;
+  --color-greywhite: #d7d6d6;
```

Comment:
did you checked impact of chaning the color, like wherever this has been used, it can break those feature(if not checked).

Reply:
it is used only here. not any other places